plugins/rapid7_insightidr/komand_rapid7_insightidr/util/endpoints.py

```diff
@@ -66,7 +66,7 @@ class QueryLogs:
     @staticmethod
     def get_query_logs(console_url: str, log_id: str):
         """
-        URI for adding get_query_logs
+        URI for adding get_query_logsœ
```

Comment:
```suggestion
        URI for adding get_query_logs
```

Reply:
Year I have no idea how that happened, thanks